wpengine/lostromos
# Add status endpoint, and move integration tests to python.

Signed-off-by: Derek Rushing <derek.rushing21@gmail.com>

# What Are We Doing Here

1. Adding a /status endpoint to show that the service is up and running.
2. Moving our integration tests to python for better manageability and a more correct implementation overall. Also adding python linting to the project.
3. Adding a total events counter to the prometheus output so that we can keep track not just of the current state counts, but also how many events have been processed in total. This also helps in testing since that is the way we can accurately track when Lostromos processed an event.
4. Adding documentation on situations where TravisCI can become slow due to a backed up job queue.

## How to Verify

1. Check out this PR, run Lostromos, and `curl localhost:8080/status` to see status output. Integration testing can be run by installing nosetests via the requirements.txt, and running `make integration-tests`.
